Jenkinsfile-testing

```diff
@@ -0,0 +1,18 @@
+pipeline {
+  agent none
+
+  stages {
+    stage("Publish package") {
+        agent {
+            ecs {
+                inheritFrom "npm"
+            }
+        }
+        steps {
+            checkout scm
+            sh 'npm ci'
+            sh 'npm test'
```

Comment:
Should we run the linter/prettier as well?

Reply:
I've set it to run eslint which runs prettier with it.